cmd/internal/cli/build.go

```diff
@@ -105,7 +105,7 @@ var buildArchFlag = cmdline.Flag{
 	Value:        &buildArgs.arch,
 	DefaultValue: runtime.GOARCH,
 	Name:         "arch",
-	Usage:        "architecture for remote build",
+	Usage:        "architecture for {obsolete}",
```

Comment:
Could we please remove use of the --arch flag instead?  Otherwise the Usage should be "obsolete" and if somebody tries to use it it should probably print a deprecated warning or an error.

Reply:
re: `remove use of the --arch flag instead` -- that may break existing users with no explanation.

1. it is not `deprecated` ... which implies "soon to be removed", rather than "removed"
2. in light of existing scripts, should error exit with appropriate error